packages/react-components/react-tree-preview/stories/Tree/TreeActions.stories.tsx

```diff
@@ -1,3 +1,8 @@
+/** @jsxRuntime classic */
+/** @jsx createElement */
+/** @jsxFrag React.Fragment */
```

Comment:
I don't think that it's a good to use slots for an example. Do customers need it for their use case? Can we avoid their usage in this story?